styles.css

```diff
@@ -0,0 +1,65 @@
+body {
+  background-image: linear-gradient(125deg, rgba(82, 48, 114, 0.9), rgba(108, 84, 194, 0.8)), url(https://media.bitdegree.org/storage/media/images/2018/08/what-is-a-web-developer.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+#container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.header {
+  text-align: center;
+  font-size: 20px;
+  color: white;
+}
+
+#survey-form {
+  padding: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: space-between;
+
+  /* padding: 50px; */
```

Comment:
Don't leave comments in your code

Reply:
I think it would be better to add more explanation to your comments and show the exact line of code that needs a change. These will help students to get where is the problem, and why did you request changes on that point and it makes them inspired to get more information about the issue.